Summary/analyze/MethodLinker/matcher.py

```diff
@@ -87,7 +87,7 @@ def get_source_files(repo_name, repo_path):
     # Only look at the top level directory in this loop
     for root, dirs, files in os.walk(repo_path):
         for dir_name in dirs:
-            if dir_name.lower() == "src" or dir_name == repo_name:
+            if dir_name.lower() == "src" or dir_name.lower() == repo_name.lower():
```

Comment:
what was the example that failed before?

Reply:
in the documented example, the directory name and repo name were both `requests` so there was no mismatch in capitalization or am I missing something?

Reply:
The library name is "Requests" but in the repo, the directory name is "requests"